backends/tfhe-cuda-backend/cuda/include/helper_multi_gpu.h

```diff
@@ -56,87 +59,113 @@ struct CudaStreams {
   cudaStream_t const *_streams;
```

Comment:
It would have been better to keep a 2D array of streams for clarity. or  `std::vector< cudaStream_t const *_streams>` which adds only one element in the FFI constructor